CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 4.0.0
+
+The Sentry FullStory integration has been updated to be compatible with Sentry JavaScript SDK 9.x. If you need to support Sentry JavaScript SDK 8.x or lower, please use version 3.x of this integration.
```

Comment:
technically, 3.x only supports v8 of the Sentry SDK. If you try to use it with an earlier version I think you'll run into this: https://github.com/getsentry/sentry-fullstory/issues/97

Reply:
Good catch - fixed with 498a18d31b7c3228dbe4fecc674fe834aec56701